KPI/J.adoc

```diff
@@ -1,54 +1,82 @@
-== J. Broken links
+== J. Links health
 
-Detection of broken links contributing to the overall quality score.
+=== Status
 
-=== J.1 To which WMCP element(s) does it apply?
+Draft
 
-All potential linked information (URLs).
-Links are generally present in <gmd:onlineResource>, <gmd:graphicOverview>, 
+=== WMCP element(s)
 
-....
-<gmd:graphicOverview>
-        <gmd:MD_BrowseGraphic>
-          <gmd:fileName>
-            <gco:CharacterString>https://navigator.eumetsat.int/preview/0deg-snow.jpg</gco:CharacterString>
-          </gmd:fileName>
-        </gmd:MD_BrowseGraphic>
-</gmd:graphicOverview>
-....
+Any element or attribute content with linked information (URLs). Links are
+generally present in `gmd:URL`, `gmd:graphicOverview`, `gmx:Anchor/@xlink:href`,
+and `@codeList` elements/attributes.
 
-....
+```xml
 <gmd:CI_OnlineResource>
    <gmd:linkage>
       <gmd:URL>https://eumetview.eumetsat.int/mapviewer/?product=EO:EUM:DAT:MSG:SNOW</gmd:URL>
    </gmd:linkage>
 </gmd:CI_OnlineResource>
-....
+```
 
-=== J.2 What is being measured?
+```xml
+<gmd:graphicOverview>
+  <gmd:MD_BrowseGraphic>
+    <gmd:fileName>
+      <gco:CharacterString>https://navigator.eumetsat.int/preview/0deg-snow.jpg</gco:CharacterString>
+    </gmd:fileName>
+  </gmd:MD_BrowseGraphic>
+</gmd:graphicOverview>
+```
 
-The number of broken links present in each individual metadata record is being measured
+```xml
+<gmd:MD_Identifier>
+  <gmd:code>
+    <gmx:Anchor xlink:actuate="onRequest" xlink:href="https://dx.doi.org/10.14287/10000004" xlink:title="DOI">doi:10.14287/10000004</gmx:Anchor>
+  </gmd:code>
+</gmd:MD_Identifier>
+```
 
-=== J.3 Why should we measure it?
+```xml
+<gmd:dateType>
+  <gmd:CI_DateTypeCode codeList="http://wis.wmo.int/2011/schemata/iso19139_2007/schema/resources/Codelist/gmxCodelists.xml#CI_DateTypeCode" codeListValue="revision" codeSpace="ISOTC211/19115">revision</gmd:CI_DateTypeCode>
```

Comment:
The codeList url is a copy of the gmxCodelists in WMO site 

Once we confirm which url are authoritatives we should cross check all codelists URL through all KPI's and give consistent examples.